CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.21.1
+------------------------------
+*December 20, 2022*
+
+### Fixed
+- CI failure on `main` due to execution of visual review PR comment step.
```

Comment:
can you elaborate on this a bit please? Having a hard time understanding what the issue was!

Reply:
Here ya go - https://github.com/justeattakeaway/pie/actions/runs/3739329333/jobs/6346777283

Currently `main` builds fail due to the Percy comment reminder you can see above.

Obviously when we merge to master, there's no associated PR, so the step falls over :)